test/powershell/Modules/Microsoft.PowerShell.Utility/Select-Object.Tests.ps1

```diff
@@ -244,6 +244,18 @@ Describe "Select-Object DRT basic functionality" -Tags "CI" {
 		$results[0] | Should -BeExactly "3"
 	}
 
+    It "Select-Object with SkipIndex should work" {
+        $results = "1", "2", "3" | Select-Object -SkipIndex 0, 2
+        $results.Count | Should -Be 1
```

Comment:
Currently we use new syntax:
```powershell
$results | Should -HaveCount 1
```

Below too.

Reply:
Didn't know. Fixing.